packages/client/hmi-client/README.md

```diff
@@ -9,6 +9,19 @@
 
 - [VS Code](https://code.visualstudio.com/) + [Volar](https://marketplace.visualstudio.com/items?itemName=Vue.volar)
 
+## HTTP Requests
+
+You can import and use the singleton [axios](https://axios-http.com/) instance to make requests to the `hmi-server` package like so:
+
+```ts
+import API from '@/api/api';
+
+// ... in an async function:
+const response = await API.get('models');
+```
```

Comment:
Instead of importing it on every page we need, can we make it a global in our single Vue app instance?

Reply:
I'm concerned that we'll regret getting in the habit of adding global variables. This way we can statically detect which files are making API calls, and any developer that sees an API call being made can easily trace back to where the axios instance was created, rather than just having to know that there's a magic global keyword floating around that you can use.

If you feel strongly about this Im happy to add the global, just wanted to throw in my 2 cents. Let me know what you think.